integration/network/ipvlan_test.go

```diff
@@ -0,0 +1,543 @@
+package network
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/network"
+	dclient "github.com/docker/docker/client"
+	"github.com/docker/docker/integration-cli/daemon"
+	"github.com/docker/docker/integration/internal/container"
+	"github.com/gotestyourself/gotestyourself/assert"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"golang.org/x/net/context"
+)
+
+func TestDockerNetworkIpvlanPersistance(t *testing.T) {
+	// verify the driver automatically provisions the 802.1q link (di-dummy0.70)
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
Commented on another PR; we should go over the tests and be consistent (either "!= linux" or "== windows").

No need to fix for this PR, as it's a general thing to look at